src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -30,26 +29,31 @@ const styles = {
 const groupCallStatuses = _.flow(
   _.values,
   _.flattenDepth(1),
-  _.countBy(a => collapseCromwellExecutionStatus(a.executionStatus))
+  _.countBy(a => {
+    const collapsedStatus = collapseCromwellStatus(a.executionStatus, a.backendStatus)
+    return collapsedStatus !== statusType.unknown ? collapsedStatus.id : collapsedStatus.label(a.executionStatus)
+  })
 )
 
 const statusCell = ({ calls }) => {
-  const { succeeded, failed, running, submitted, ...others } = groupCallStatuses(calls)
+  const statusGroups = groupCallStatuses(calls)
+  // Note: these variable names match the id values of statusType (except for unknownStatuses, which will be their labels).
+  const { succeeded, failed, running, submitted, waitingForQuota, ...unknownStatuses } = statusGroups
 
-  const makeRow = (count, icon, text) => {
+  const makeRow = (count, status, labelOverride) => {
+    const seeMore = !!status.moreInfoLink ? h(Link, { href: status.moreInfoLink, style: { marginLeft: '0.50rem' }, ...Utils.newTabLinkProps },
+      [status.moreInfoLabel, icon('pop-out', { size: 12, style: { marginLeft: '0.25rem' } })]) : ''
     return !!count && div({ style: { display: 'flex', alignItems: 'center', marginTop: '0.25rem' } }, [
-      icon,
-      ` ${count} ${text}`
+      status.icon(),
+      ` ${count} ${!!labelOverride ? labelOverride : status.label()}`,
+      seeMore
     ])
   }
-
-  return h(Fragment, [
-    makeRow(submitted, submittedIcon(), 'Submitted'),
-    makeRow(running, runningIcon(), 'Running'),
-    makeRow(succeeded, successIcon(), 'Succeeded'),
-    makeRow(failed, failedIcon(), 'Failed'),
-    _.map(([name, count]) => makeRow(count, unknownIcon(), name), _.toPairs(others))
-  ])
+  return h(Fragment, _.concat(
+    [statusType.submitted.id, statusType.waitingForCloudQuota.id, statusType.running.id, statusType.succeeded.id, statusType.failed.id].filter(
```

Comment:
@dmohs had you thought of a way to do this that got rid of the need for the decomposition entirely (given "unknownStatus", formerly known as "others")? I wasn't clear from your comment, and this was the best I could come up with. I still think it's an improvement over what I had previously.

FWIW, here's an example with "unknownStatuses":

![image](https://user-images.githubusercontent.com/484484/152406952-3612ebb6-50c5-48af-96c7-9b0ab23dd73c.png)